tests/test_home.py

```diff
@@ -25,3 +26,73 @@ def client():
 def test_home(client):
     rv = client.get('/')
```

Comment:
I'm not sure why it's `rv` here, Do you think it would be better to rename it to `response` everywhere?